app/ocr.py

```diff
@@ -521,7 +515,7 @@ def get_matches(input_list, topic, full_text):
         gang_match = get_matches(gang_list, 'Gang', self.doc)
 
         # Printing full_text[judge_match2[0][1]:judge_match2[0][2]] gives word
-        # it matches on, can put in the [0] a for loop to see all matches
+        # it matches on, can put in the [0] a for loop to see all matche78s
```

Comment:
this was an accident but I dont want to completely repush everything for this

Reply:
Best thing to here is to simply make that quick fix on your local, then run: `git add ocr.py` -> `git commit -m 'fixes typo on line 518'` -> `git push origin remove_decision_type`. This will push only this small fix to this exact pull request!

Reply:
thank you :)
12345